Final_project/process_sales.py

```diff
@@ -0,0 +1,66 @@
+"""
+ Process sales in the final project
+"""
+
+import datetime as dt
+
+from airflow import DAG
+from airflow.providers.google.cloud.operators.bigquery import BigQueryInsertJobOperator
+from table_defs.sales_csv import sales_csv
+
+DEFAULT_ARGS = {
+    'depends_on_past': True,
```

Comment:
Actually, you should use False here because the data is partitioned, and each day is independent. If you use True, you cannot run each DAG run in parallel.

Reply:
If you use MERGE somewhere in your pipeline, you should set depends_on_past to True, of course